components/footer/favs/favsBtn.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+/* import { Story, Meta } from "@storybook/react/types-6-0";
+import favsBtn from "./favs.png";
+
+export default {
+  title: "favsBtn",
+  component: favsBtn,
+} as Meta;
+
+const image = {
+  src: favsBtn,
+  alt: "add-to-favs",
+};
+
+export const favsImg = () => <img src={image.src} alt={image.alt} />;
+ */
```

Comment:
If you don't need this code, better remove it from this pull request :) 
Same with the HomeBtn